python/ray/train/examples/mosaic_cifar10_example.py

```diff
@@ -37,13 +37,13 @@ def trainer_init_per_worker(config):
         datasets.CIFAR10(
             data_directory, train=True, download=True, transform=cifar10_transforms
         ),
-        list(range(64)),
+        list(range(BATCH_SIZE * 10)),
     )
     test_dataset = torch.utils.data.Subset(
         datasets.CIFAR10(
             data_directory, train=False, download=True, transform=cifar10_transforms
         ),
-        list(range(64)),
+        list(range(2048)),
```

Comment:
why is batch size for training const and this inline number ?

Reply:
Updated so that it is `BATCH_SIZE *10` just like the train dataset